packages/common/src/store/upload/selectors.ts

```diff
@@ -20,31 +20,62 @@ const TRANSCODE_WEIGHT = 1 - UPLOAD_WEIGHT
 const AUDIO_WEIGHT = 1
 const ART_WEIGHT = 0
 
+/**
+ * Get the upload and transcode status of a track including its stems.
+ * Results in [0,1] bounded progress for each type (fileUpload and transcode).
+ */
+const trackProgressSummary = (
```

Comment:
unit test for this would be nice

Reply:
you betcha, planning to add tests galore for this flow